src/core/db/backend/pg.py

```diff
@@ -764,3 +768,158 @@ def import_file_w_dbtruck(self, table_name, file_path):
 
         return import_datafiles([file_path], create_new, table_name, errfile,
                                 PGMethods, **dbsettings)
+
+    def escape_quotes(self, parameter):
```

Comment:
It doesn't seem like anything outside of pg.py uses `escape_quotes`, so it should probably be marked as private.